docs/application/toc_application.md

```diff
@@ -416,6 +416,7 @@
 ####### [Watch Application](/application/native/guides/applications/dali-watch-app.md)
 ####### [Widget Application](/application/native/guides/applications/dali-widget-app.md)
 ##### [Service Applications](/application/native/guides/applications/service-app.md)
+##### [Component-based Applications](/application/native/guides/applications/component-based-app.md)
```

Comment:
Thank you for updating the TOC file. 
docs/application/toc_all.md is the right file. 
This toc_application.md file is only for back up.

For more information, see https://github.com/Samsung/tizen-docs/pull/805#discussion_r314559172